src/__tests__/directLine.test.ts

```diff
@@ -0,0 +1,19 @@
+import * as DirectLineExport from '../directLine';
+
+test('#setConnectionStatusFallback', () => {
+    const { DirectLine } = DirectLineExport;
+    expect(typeof DirectLine.prototype.setConnectionStatusFallback).toBe('function')
+    const { setConnectionStatusFallback } = DirectLine.prototype
+    const testFallback = setConnectionStatusFallback(0, 1)
+    let idx = 4
+    while(idx--) {
```

Comment:
Space before parenthesis.

```suggestion
    while (idx--) {
```